src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    operate(callback, number) {
+      if (typeof callback === `function`) {
```

Comment:
Consider adding an `else` block to handle cases where `callback` is not a function. This could involve throwing an error or returning a specific value to indicate improper usage.